web/viewer.css

```
@@ -71,6 +71,8 @@
   --loading-icon: url(images/loading.svg);
   --treeitem-expanded-icon: url(images/treeitem-expanded.svg);
   --treeitem-collapsed-icon: url(images/treeitem-collapsed.svg);
+  --toolbarButton-editorFreeText-icon: url(images/toolbarButton-editorFreeText.svg);
+  --toolbarButton-editorSelect-icon: url(images/toolbarButton-editorSelectTool.svg);
```

```suggestion
  --toolbarButton-editorNone-icon: url(images/toolbarButton-editorNone.svg);
```

and please move this up one line.